app/scripts/modules/amazon/src/loadBalancer/configure/AmazonLoadBalancerChoiceModal.tsx

```diff
@@ -87,6 +88,23 @@ export class AmazonLoadBalancerChoiceModal extends React.Component<
     return false;
   }
 
+  // This assumes that () and [] only exist in the link markup, and satisfies the use cases of this component.
+  private parseMessageForLinks(message: string) {
```

Comment:
Open to using a markdown parser, but most use cases in this component will only be simple strings. The first implementation happens to have a link with it.

Reply:
I would definitely prefer to use `<Markdown message={message} />` here

Reply:
Perfect, didn't see that this existed. I will swap it out.